### PR TITLE
Touch event

### PR DESCRIPTION
**Pull request by zcw90**
Repair mDisableCircularTransformation to true, the Touch event is invalid.

### CODE REVIEW

File: circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java [hunk 1]
```diff
@@ -418,6 +418,10 @@ public boolean onTouchEvent(MotionEvent event) {
     }
 
     private boolean inTouchableArea(float x, float y) {
+        // mDisableCircularTransformation is true, handle Touch events normally.
```
Comment (hdodenhof) [inline circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java]: I think this is obvious and doesn't need a comment.

File: circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java [hunk 2]
```diff
@@ -418,6 +418,10 @@ public boolean onTouchEvent(MotionEvent event) {
...
+        if(mDisableCircularTransformation) {
```
Comment (hdodenhof) [inline circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java]: There should be a space after the `if`.

File: circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java [hunk 3]
```diff
@@ -418,6 +418,10 @@ public boolean onTouchEvent(MotionEvent event) {
...
+            return true;
+        }
```
Comment (hdodenhof) [inline circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java]: There should be an empty line after this line.

File: circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java (hunk 2 above)
Comment (zcw90) [reply circleimageview/src/main/java/de/hdodenhof/circleimageview/CircleImageView.java]: Yeah,such is the case。

**Comment by zcw90** [comment]
Repair issues #325 .

**Comment by hdodenhof** [comment]
This has been taken care of in e0e4384ed030afc0353998af676f2988eb9fea2a and c109651db74fcb94ed57ed85e721384ed6535d2b. Thanks for the initial idea!

**Pull request closed without merging**